tests/test_iam_manager.py

```diff
@@ -39,3 +39,12 @@ def test_role_arn(im):
 
     # Then
     assert arn is not None
+
+def test_get_policies(im):
```

Comment:
Let's apply `Given When Then` format to those.

Reply:
sure I will take that format